From f8d84d9cb1f3ecfef54ed803d94e4a73a30a626b Mon Sep 17 00:00:00 2001
From: Biwen Li <biwen.li@nxp.com>
Date: Fri, 9 Aug 2019 15:10:29 +0800
Subject: [PATCH 030/245] Revert "arm: defconfig: enable Soc Drivers on
 LS1021A"

commit 7b8f804fb79e0d31b3c1cb904348d00d10ea9fdd from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

This reverts commit 289b5a5ce1d9fac494401c1c7ba379778c184f55.

Revert "soc: fsl: add support for arm64/Layerscape SOC specific drivers"

This reverts commit 600f0e8656c2bc361f1a25a452661d0165964e6e.

Fix some conflict files:
	drivers/soc/fsl/Makefile
	driver/soc/fsl/Kconfig

Dest:
	drop old flextimer driver

Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/soc/fsl/Kconfig             |  4 ----
 drivers/soc/fsl/Kconfig.arm         | 16 ----------------
 drivers/soc/fsl/Makefile            |  1 -
 drivers/soc/fsl/layerscape/Kconfig  | 10 ----------
 drivers/soc/fsl/layerscape/Makefile |  1 -
 5 files changed, 32 deletions(-)
 delete mode 100644 drivers/soc/fsl/Kconfig.arm
 delete mode 100644 drivers/soc/fsl/layerscape/Kconfig
 delete mode 100644 drivers/soc/fsl/layerscape/Makefile

diff --git a/drivers/soc/fsl/Kconfig b/drivers/soc/fsl/Kconfig
index 03c9bd2a6ae3..bc809f4ca58f 100644
--- a/drivers/soc/fsl/Kconfig
+++ b/drivers/soc/fsl/Kconfig
@@ -34,7 +34,3 @@ config FSL_SLEEP_FSM
 	  This driver configures a hardware FSM (Finite State Machine) for deep sleep.
 	  The FSM is used to finish clean-ups at the last stage of system entering deep
 	  sleep, and also wakes up system when a wake up event happens.
-
-if ARM || ARM64
-source "drivers/soc/fsl/Kconfig.arm"
-endif
diff --git a/drivers/soc/fsl/Kconfig.arm b/drivers/soc/fsl/Kconfig.arm
deleted file mode 100644
index 106c9b981ac1..000000000000
--- a/drivers/soc/fsl/Kconfig.arm
+++ /dev/null
@@ -1,16 +0,0 @@
-#
-# Freescale ARM SOC Drivers
-#
-
-config LS_SOC_DRIVERS
-	bool "Layerscape Soc Drivers"
-	depends on ARCH_LAYERSCAPE || SOC_LS1021A
-	default n
-	help
-	Say y here to enable Freescale Layerscape Soc Device Drivers support.
-	The Soc Drivers provides the device driver that is a specific block
-	or feature on Layerscape platform.
-
-if LS_SOC_DRIVERS
-	source "drivers/soc/fsl/layerscape/Kconfig"
-endif
diff --git a/drivers/soc/fsl/Makefile b/drivers/soc/fsl/Makefile
index 104c68fbe43e..3c062e951f5f 100644
--- a/drivers/soc/fsl/Makefile
+++ b/drivers/soc/fsl/Makefile
@@ -9,5 +9,4 @@ obj-$(CONFIG_FSL_QIXIS) 		+= qixis_ctrl.o
 obj-$(CONFIG_FSL_GUTS)			+= guts.o
 obj-$(CONFIG_FSL_LS2_CONSOLE)		+= ls2-console/
 obj-$(CONFIG_SUSPEND)			+= rcpm.o
-obj-$(CONFIG_LS_SOC_DRIVERS)		+= layerscape/
 obj-$(CONFIG_FSL_SLEEP_FSM)	+= sleep_fsm.o
diff --git a/drivers/soc/fsl/layerscape/Kconfig b/drivers/soc/fsl/layerscape/Kconfig
deleted file mode 100644
index e1373aa1b4ea..000000000000
--- a/drivers/soc/fsl/layerscape/Kconfig
+++ /dev/null
@@ -1,10 +0,0 @@
-#
-# Layerscape Soc drivers
-#
-config  FTM_ALARM
-	bool "FTM alarm driver"
-	default n
-	help
-	Say y here to enable FTM alarm support.  The FTM alarm provides
-	alarm functions for wakeup system from deep sleep.  There is only
-	one FTM can be used in ALARM(FTM 0).
diff --git a/drivers/soc/fsl/layerscape/Makefile b/drivers/soc/fsl/layerscape/Makefile
deleted file mode 100644
index 6299aa1de215..000000000000
--- a/drivers/soc/fsl/layerscape/Makefile
+++ /dev/null
@@ -1 +0,0 @@
-obj-$(CONFIG_FTM_ALARM) += ftm_alarm.o
-- 
2.17.1

